app/main/views.py

```diff
@@ -185,6 +196,12 @@ def get_service_by_id(service_id):
 
 
 @main.route('/search')
+@deprecated(dies_at=datetime(2016, 1, 1))
+def redirect_search():
+    return redirect(url_for(".search", **request.args), code=301)
```

Comment:
Cool, does this pass along all arguments included in the original request?  Does it do GET and POST arguments?

I was looking for something like this the other day to use with the supplier app.